modules/openy_features/openy_node/modules/openy_node_landing/openy_node_landing.info.yml

```diff
@@ -1,21 +1,22 @@
-name: Open Y Node Landing
-description: Provides a Landing page content type.
+name: 'Open Y Node Landing'
+description: 'Provides a Landing page content type.'
 type: module
 core: 8.x
-package: Open Y
+package: 'Open Y'
 version: 8.x-1.0
 dependencies:
-  - datalayer:datalayer
-  - entity_reference_revisions:entity_reference_revisions
-  - drupal:field
-  - field_group:field_group
-  - drupal:menu_ui
-  - metatag:metatag
-  - drupal:node
-  - openy_node:openy_node
-  - drupal:options
-  - paragraphs:paragraphs
-  - drupal:path
-  - pathauto:pathauto
-  - scheduler:scheduler
-  - drupal:user
+  - 'datalayer:datalayer'
+  - 'drupal:field'
+  - 'drupal:menu_ui'
+  - 'drupal:node'
+  - 'drupal:options'
+  - 'drupal:path'
+  - 'drupal:user'
+  - 'entity_reference_revisions:entity_reference_revisions'
+  - 'field_group:field_group'
+  - 'metatag:metatag'
+  - 'openy_node:openy_node'
+  - 'paragraphs:paragraphs'
+  - 'pathauto:pathauto'
+  - 'scheduler:scheduler'
+core_incompatible: false
```

Comment:
I'm not sure about this one, looks like a bug in Features module - https://www.drupal.org/project/features/issues/3122684
Let's remove it.